src/index.html

```diff
@@ -13,5 +13,153 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <fieldset>
+      <legend class="blockname">Personal info</legend>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+        <div class="form_field">
+          <label>
+            Surename:
+            <input type="text"
+                   name="UserLastName"
+                   required>
```

Comment:
```suggestion
            <input 
              type="text"
              name="userLastName"
              required
            >
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/232299396-efce1c45-8215-410a-bad2-3c7ca37abc3d.png)
![image](https://user-images.githubusercontent.com/96409563/232299436-7017f1c3-88aa-495a-b431-163d7c479989.png)